RaySeptian_211011401233/funtional.docx

```diff
@@ -0,0 +1,8 @@
+Meja:
+Pelanggan bisa memilih menu
+Menentukan jumlah pesanan
+Memilih jenis pembayaran
+Memilih dine in atau take home
+Koki:
+Memberikan informasi tersedianya menu
+Memberi tahu stok makanan
```

Comment:
Good job, but you can do more than this.
Terlalu singkat dan belum menjelaskan secara rinci apa yang dibutuhkan oleh user, sehingga akan menyulitkan proses untuk analisis dan pengembangan software ke depannya.